feature-libs/product-configurator/rulebased/root/http-interceptors/configurator-bad-request.handler.ts

```diff
@@ -46,7 +46,12 @@ export class ConfiguratorBadRequestHandler extends HttpErrorHandler {
 
   protected handleIllegalArgumentIssues(message: string): void {
     if (this.isIllegalStateErrorRelatedToMakeToStock(message)) {
-      this.globalMessageService.add(message, GlobalMessageType.MSG_TYPE_ERROR);
+      this.globalMessageService.add(
+        {
+          key: 'configurator.httpHandlers.makeToStockBaseProductIssue',
+        },
+        GlobalMessageType.MSG_TYPE_ERROR
+      );
```

Comment:
wouldn't a test make sense here -  so checking that we call the message service with the proper key instead of the exception text.

Reply:
Good point! I extended the test accordingly.